bin/juxtaposer/deploy/juxtaposer_deployment_template.yml

```diff
@@ -0,0 +1,111 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ${APP_SERVICE_ACCOUNT}
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  labels:
+    app: ${APP_NAME}
+  name: ${APP_NAME}
+  namespace: ${APP_NAMESPACE}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ${APP_NAME}
+  template:
+    metadata:
+      labels:
+        app: ${APP_NAME}
+    spec:
+      serviceAccountName: ${APP_SERVICE_ACCOUNT}
+
+      volumes:
+        - name: ${APP_NAME}-secretless-config
+          configMap:
+            name: ${APP_NAME}-secretless-config
+            defaultMode: 420
+        - name: ${APP_NAME}-config
+          configMap:
+            name: ${APP_NAME}-config
+            defaultMode: 420
+        - name: ${APP_NAME}-sockets
+          emptyDir:
+            medium: Memory
+
+      containers:
+      - name: ${APP_NAME}
+        image: ${PERFTOOL_IMAGE}
+        imagePullPolicy: Always
+        # command: [ "/bin/sleep", "999d" ]
+        args: ["-c", "-t", "${TEST_DURATION}", "-f", "/etc/${APP_NAME}/${APP_NAME}_${CONFIG_TEMPLATE}.yml"]
+
+        volumeMounts:
+          - mountPath: /etc/${APP_NAME}
+            name: ${APP_NAME}-config
+            readOnly: true
+          - mountPath: /sock
+            name: ${APP_NAME}-sockets
+            readOnly: false
+      - name: secretless
+        image: cyberark/secretless-broker:latest
```

Comment:
can we have another image as a param? `cyberark/secretless-broker:0.7.1-becf48d` is the image from the latest commit on master - it would be great to run some tests against that with the v2 config

Reply:
sure